lib/amqp_client.js

```diff
@@ -192,18 +192,22 @@ AMQPClient.prototype.createSender = function(address, options) {
     throw new Error('Must connect before creating links');
   }
 
-  address = address || this._defaultQueue;
+  address = u.parseLinkAddress(address || this._defaultQueue);
```

Comment:
I wasn't sure what the EH/SB spec was for this. If slashes aren't allowed for queue names this should "just work TM"